main/integration-tests/config/settings/example.yml

```diff
@@ -0,0 +1,14 @@
+# Enter the Test Administrator username and password below. (Please make sure this user has admin privileges)
+username: ""
+password: ""
+
+# This is a Project ID in your SWB deployment safe to be used by integration tests
```

Comment:
Does this comment need to be rephrased? It looks like it's a run on sentence.

Reply:
Fixed. Coming in the next commit